### ISSUE TITLE
Client unit tests suck "loading", all others work.

### ISSUE THREAD

**Issue by luisatlive** [Sanjo/meteor-jasmine#197]
I've gotten client integration and server unit and integration tests working, but the client unit tests sit forever in the loading stage. Karma is blank, when I click on the debug I get a page that just says:

Couldn't find a template named "layout" or "layout". Are you sure you defined it?

The client-unit-test.log says:
Spawn script arguments:
parentPid: 40666
command: /Users/luis/.meteor/packages/meteor-tool/.1.1.3.11mjk3u++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node
commandArguments: [ '/Users/luis/Jobby-Web/.meteor/local/build/programs/server/npm/sanjo_karma/node_modules/karma/bin/karma',
  'start',
  '/Users/luis/Jobby-Web/.meteor/local/karma/jasmine-client-unit.config.js' ]
pid: 40686
child.pid: 40687
ESC[32mINFO [karma]: ESC[39mKarma v0.12.31 server started at http://localhost:9876/
ESC[32mINFO [launcher]: ESC[39mStarting browser Chrome
ESC[33mWARN [watcher]: ESC[39mPattern "/Users/luis/Jobby-Web/.meteor/local/build/programs/server/assets/packages/sanjo_jasmine/src/package/node_modules/component-mocker/index.js" does not match any file.
ESC[33mWARN [watcher]: ESC[39mPattern "/Users/luis/Jobby-Web/tests/jasmine/client/unit/**/*-{stub,stubs,mock,mocks}.{js,coffee,litcoffee,coffee.md}" does not match any file.
ESC[32mINFO [Chrome 42.0.2311 (Mac OS X 10.10.2)]: ESC[39mConnected on socket iyWuvTS9bSqAEWmg6h4n with id 81026766
ESC[31mChrome 42.0.2311 (Mac OS X 10.10.2) ERRORESC[39m
  Uncaught TypeError: Cannot read property 'FastRender' of undefined
  at /Users/luis/Jobby-Web/.meteor/local/build/programs/web.browser/packages/global-imports.js:32

The line from global-imports.js that's mentioned above is:
FastRender = Package['meteorhacks:fast-render'].FastRender;

Any ideas?

**Comment by luisatlive** [comment]
Command I used:
JASMINE_CLIENT_UNIT=1 JASMINE_CLIENT_INTEGRATION=0 JASMINE_SERVER_UNIT=0 JASMINE_SERVER_INTEGRATION=0 DEBUG=1 JASMINE_DEBUG=1 VELOCITY_DEBUG=1 VELOCITY_DEBUG_MIRROR=1 ROOT_URL=http://localhost:4000 meteor --port=4000 --settings ./tests/settings/local.json

**Comment by luisatlive** [comment]
Packages:
abhiaiyer:meteor-twilio                     0.0.2  Twilio Bindings for Meteor
abhiaiyer:shorten-it                        0.0.5  Shorten It - URL Shortner
accounts-facebook                           1.0.4  Login service for Facebook accounts
accounts-google                             1.0.4  Login service for Google accounts
accounts-password                           1.1.1  Password support for accounts
accounts-ui                                 1.1.5  Simple templates to add login widgets to an app
aldeed:http                                 0.2.2  Improves the core HTTP package
anti:i18n                                   0.4.3  Internalization: simplest package
awatson1978:browser-detection               1.0.4  Meteorite package that provides browser detection.
cmather:handlebars-server                   2.0.0  Allows handlebars templates to be defined on the server in .handlebars files
coffeescript                                1.0.6  Javascript dialect with fewer braces and semicolons
deps                                        1.0.7  Deprecated: Use the 'tracker' package instead.
djedi:sanitize-html                         1.6.1  Sanitze HTML
doctorpangloss:filter-collections           0.2.9  Filter, paging and sort for Meteor collections with hooks and Iron Router s...
doctorpangloss:method-hooks                 2.0.2  Run functions before and after methods, mutate arguments and return values.
fastclick                                   1.0.3  Faster touch events on mobile
force-ssl                                   1.0.4  Require this application to use HTTPS
fourseven:scss                              1.2.3* Style with attitude and autoprefixer.
gadicohen:sitemaps                          0.0.20  functions to easily output valid sitemaps
gfk:mailgun-api                             1.1.0  Mailgun API - Mailgun api implementation for Meteor.
http                                        1.1.0  Make HTTP calls to remote servers
iron:router                                 1.0.7  Routing specifically designed for Meteor
jquery                                      1.11.3_2  Manipulate the DOM using CSS selectors
less                                        1.0.14  The dynamic stylesheet language
manuelschoebel:ms-seo                       0.4.1  Easily config SEO for your routes
meteor                                      1.1.6  Core Meteor environment
meteorhacks:aggregate                       1.2.1  Proper MongoDB aggregations support for Meteor
meteorhacks:async                           1.0.0  Set of async utilities to work with NPM modules and other async code blocks
meteorhacks:fast-render                     2.3.1* Render you app even before the DDP connection comes live. - magic?
meteorhacks:kadira                          2.20.1  Performance Monitoring for Meteor
meteorhacks:kadira-profiler                 1.1.0  CPU Profiler for Kadira
meteorhacks:subs-manager                    1.3.0  Subscriptions Manager for Meteor
meteorhacks:unblock                         1.1.0  a way to use this.unblock inside publications
momentjs:moment                             2.10.3  Moment.js (official): parse, validate, manipulate, and display dates - off...
mongo-livedata                              1.0.8  Moved to the 'mongo' package
multiply:iron-router-progress               1.0.1  Progressbar for iron:router
npm-bcrypt                                  0.7.8_2  Wrapper around the bcrypt npm package
okgrow:iron-router-autoscroll               0.0.7* Fixes page position after changing pages using Iron Router
pauli:accounts-linkedin                     1.1.2  Accounts service for LinkedIn accounts
peerlibrary:aws-sdk                         2.1.17_1  SDK for AWS services including Amazon S3, Amazon EC2, DynamoDB, and Amaz...
percolatestudio:percolatestudio-migrations  0.7.2  Allows you to define and run db migrations.
percolatestudio:synced-cron                 1.1.0  Allows you to define and run scheduled jobs across multiple servers.
raix:handlebar-helpers                      0.2.4  Handlebar helpers
random                                      1.0.3  Random number generator and utilities
reactive-var                                1.0.5  Reactive variable
sanjo:jasmine                               0.13.2  Easily use Jasmine in Meteor
service-configuration                       1.0.4  Manage the configuration for third-party services
session                                     1.1.0  Session variable
sha                                         1.0.3  SHA256 implementation
spacebars                                   1.0.6  Handlebars-like template language for Meteor
spiderable                                  1.0.7  Makes the application crawlable to web spiders
standard-app-packages                       1.0.5  Moved to meteor-platform
ui                                          1.0.6  Deprecated: Use the 'blaze' package
underscore                                  1.0.3  Collection of small helpers: _.map, _.each, ...
underscore-deep                             0.0.1+
velocity:core                               0.6.0  Velocity, a Meteor specific test-runner
velocity:html-reporter                      0.5.3  Reactive Velocity test reports in your app.
webapp                                      1.2.0  Serves a Meteor app over HTTP
workpop-npm                                 1.0.0+ Defines Workpop npm packages
workpop:push                                2.6.5  Isomorphic Push notifications for APN and GCM

**Comment by Sanjo** [comment]
I'm fixing the problem with meteorhacks:fast-render now.

**Comment by luisatlive** [comment]
Perfect; that was quick :+1:

**Comment by Sanjo** [comment]
The error with meteorhacks:fast-render is fixed in version 0.13.3. Let me know if you have other issues.

**Comment by luisatlive** [comment]
Updating now...

**Comment by luisatlive** [comment]
Worked perfectly; now the sample tests run and pass... posting a separate issue on errors I'm seeing in the logs but which don't seem to stop tests from passing.

**Comment by Sanjo** [comment]
Ok, great!
